tasks/measurements/alexa/alexa.py

```diff
@@ -0,0 +1,111 @@
+from typing import Dict
+from netunicorn.base import Task
+from pprint import pprint
+import subprocess
+from ping3 import ping
+import csv
+import json
+
+class AlexaWebsitesTask(Task):
+    # Measure network metrics for a list of Alexa top websites.
+    requirements = [
+        "sudo apt-get install -y curl dnsutils traceroute",
+        "pip install ping3"
+    ]
+
+    def __init__(self, domain: str = None, filepath: str = None, output_path: str = None, top_k: int = None):
```

Comment:
please, accept the *args and **kwargs and pass them to the base class, some important things may be passed there (e.g., task name)